Cargo.toml

```diff
@@ -211,6 +211,7 @@ project-root = "0.2.2"
 rayon = "1.10.0"
 ropey = "1.6.1"
 rust-lapper = "1.1.0"
+rustversion = "1.0.20"
```

Comment:
I chose the version which was listed in `Cargo.lock`, so it does not change any other behavior.

Reply:
```suggestion
```